conanfile.py

```diff
@@ -129,9 +136,12 @@ def configure(self):
 
     def build(self):
         cmake = CMake(self)
-        cmake.definitions["WITH_FUZZING"] = "ON" if self.options.with_fuzzing else "OFF"
         cmake.definitions["WITH_GUI"] = "ON" if self.options.with_gui else "OFF"
-        cmake.definitions["CRASHDUMP_SERVER"] = self.options.crashdump_server
+        if self.options.with_gui:
```

Comment:
Maybe in this case, it would be easier to read if we use 
```
if self.options.with_gui:
  if self.options.with_crash_handling:
    cmake.definitions["WITH_CRASH_HANDLING"] = "ON"
    cmake.definitions["CRASHDUMP_SERVER"] = self.options.crashdump_server
  else:
    cmake.definitions["WITH_CRASH_HANDLING"] = "OFF"
```
as both cmake options depends on conan with_crash_handling option

Reply:
Great idea! Thanks! will fix it...

Reply:
done.